packages/analytics/add-global-link-handler/index.ts

```diff
@@ -0,0 +1,86 @@
+import { getProductIntentFromURL } from '../get-product-intent-from-url'
+import { getUTMParamsCaptureState } from '../utm-params-capture'
+
+const destinations: string[] = [
+  'https://app.terraform.io',
+  'https://boundaryproject.io',
+  'https://cloud.hashicorp.com',
+  'https://consul.io',
+  'https://developer.hashicorp.com',
+  'https://hashicorp.com',
+  'https://packer.io',
+  'https://portal.cloud.hashicorp.com',
+  'https://terraform.io',
+  'https://vaultproject.io',
+  'https://waypointproject.io',
+]
+
+const containsDestination = (str: string): boolean =>
+  destinations.some(function (destination) {
+    return str.indexOf(destination) >= 0
+  })
+
+// Track if we've setup this handler already to prevent registering the handler
+// multiple times.
+let hasHandler = false
+
+export function addGlobalLinkHandler(
+  callback?: (destinationUrl: string) => void
+) {
+  if (typeof window === 'undefined' || hasHandler) return
+
+  window.addEventListener('click', (event) => {
+    const linkElement = (event.target as HTMLElement).closest('a')
+
+    if (linkElement && containsDestination(linkElement.href)) {
+      const segmentAnonymousId = getSafeSegmentId()
+      const productIntent = getProductIntentFromURL()
+      const utmParams = getUTMParamsCaptureState()
+
+      event.preventDefault()
+
+      const url = new URL(linkElement.href)
+
+      if (segmentAnonymousId) {
+        url.searchParams.set('ajs_uid', segmentAnonymousId)
```

Comment:
```suggestion
        url.searchParams.set('ajs_aid', segmentAnonymousId)
```

I think this should be `ajd_aid`, not `ajs_uid` to set the right value on the receiving end.

Reply:
whoops, good catch!

Reply:
https://github.com/hashicorp/web-platform-packages/pull/103/commits/8a5b2a9c4af5cdae2ad99c920fe97a87e573b823